### ISSUE TITLE
Bug: v6.2.1 does not build

### ISSUE THREAD

**Issue by Lyokolux** [vueuse/vueuse#721]
I just installed your package v6.2.1 since you released it 9 minutes ago.

When I try to use the version v6.2.1, Webpack returns the following error:
>ModuleDependencyError: export 'effectScope' (imported as 'effectScope') was not found in 'vue-demi' 

Installing the previous version (v6.2.0) make it works. So I suppose it is a regression.

**Comment by antfu** [comment]
It doesn't seem to be anything related to the build, can you share a minimal reproduce repo?

**Comment by Mootook** [comment]
Not sure if it's related, but I faced issues when upgrading my app to use 6.2.x from 6.1.0.

```

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'bypassFilter' from non EcmaScript module (only default export is available)

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'clamp' from non EcmaScript module (only default export is available)

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'computed' from non EcmaScript module (only default export is available)

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'containsProp' from non EcmaScript module (only default export is available)

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'createEventHook' from non EcmaScript module (only default export is available)

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'createFilterWrapper' from non EcmaScript module (only default export is available)

 error  in ./node_modules/@vueuse/core/index.mjs

Can't import the named export 'createSingletonPromise' from non EcmaScript module (only default export is available)
```

**Comment by chuckntaylor** [comment]
Exact same sort of output for me as what @Mootook reported above. Rolling back to v6.1.0 allowed me to build again.

**Comment by antfu** [comment]
If you mean v6.1 between v6.2, and you are using Webpack, see #637

**Comment by FRSgit** [comment]
Hey @Lyokolux @Mootook @chuckntaylor! This comment should help you out - https://github.com/vueuse/vueuse/issues/718#issuecomment-913319680 (at least that's what've worked for me)

**Comment by chuckntaylor** [comment]
Hey everyone, thanks for this.
The problems came up for me when I was adding PWA to an existing project and this package was updated at that time. I found that if I moved to the CLI version 5 beta,
```json
"@vue/cli-service": "^5.0.0-beta.3",
"@vue/cli-plugin-pwa": "^5.0.0-beta.3"
````
the problem went away - even without the added webpack configuration in vue.config.js.

Still... I've just added the config that @antfu suggested for good measure.
